app/views/comments/_form.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= form_for(@comment) do |f| %>
```

Comment:
This will become `form_for(@image, @comment) do |f|` with the routing update listed below